scylla/src/statement/prepared_statement.rs

```diff
@@ -613,21 +572,22 @@ mod tests {
             ],
             [0, 3, 4],
         );
-        let pk = make_key([
-            67i8.to_be_bytes().as_ref(),
-            89i64.to_be_bytes().as_ref(),
-            &[1, 2, 3, 4, 5],
-        ]);
-        let cells = PartitionKeyDecoder::new(&meta, &pk)
-            .collect::<Result<Vec<_>, _>>()
-            .unwrap();
+        let mut values = SerializedValues::new();
+        values.add_value(&67i8).unwrap();
+        values.add_value(&42i16).unwrap();
+        values.add_value(&23i32).unwrap();
+        values.add_value(&89i64).unwrap();
+        values.add_value(&[1u8, 2, 3, 4, 5]).unwrap();
+
+        let pk = PartitionKey::new(&meta, &PartitionerName::Murmur3, &values).unwrap();
+        let pk_cols = Vec::from_iter(pk.iter().map(|(value, spec)| (value, spec)));
```

Comment:
`.map(|(value, spec)| (value, spec))`

Does this part do anything?

Reply:
It's a quite funny remnant.